devops/iam-did-auth-proxy-dev/values.yaml

```
@@ -74,14 +74,14 @@ did-auth-proxy-helm:
 
   opsValues:
     RPC_URL: https://volta-rpc.energyweb.org/
-    CACHE_SERVER_URL: https://identitycache-dev.energyweb.org/v1
+    CACHE_SERVER_URL: https://identitycache-gp4btc-dev.energyweb.org/v1
     REDIS_HOST: iam-did-auth-proxy-redis-master.iam-did-auth-proxy.svc.cluster.local
     REDIS_PORT: 6379
     REDIS_PASSWORD: redis
     IPFS_HOST: ipfs.infura.io
     IPFS_PORT: 5001
     IPFS_PROTOCOL: https
-    IPFS_PROJECTID: 2DXJGR6JFXIE612O8bUvjXGIBIP
+    IPFS_PROJECTID: 2MdfPOCDrxelBHhNLArgfuG7pmB
```

FYI: we have two IPFS projects. One is `2MdfPOCDrxelBHhNLArgfuG7pmB`, which is used in [dev](https://github.com/energywebfoundation/did-auth-proxy/blob/e382f5bd5e46fb657935d4553b29c0119ebd4577/devops/iam-did-auth-proxy-dev/values.yaml#L80) and [prod](https://github.com/energywebfoundation/did-auth-proxy/blob/e382f5bd5e46fb657935d4553b29c0119ebd4577/devops/iam-did-auth-proxy-prod/values.yaml#L80) and another one in `2MdfHcPJmw6mxgU51to2daMbGpZ` [staging](https://github.com/energywebfoundation/did-auth-proxy/blob/e382f5bd5e46fb657935d4553b29c0119ebd4577/devops/iam-did-auth-proxy-staging/values.yaml#L80)

---

@JGiter I think this change is incorrect. I think that the previous value `2DXJGR6JFXIE612O8bUvjXGIBIP` is correct and shouldn't be changed.
```suggestion
    IPFS_PROJECTID: 2DXJGR6JFXIE612O8bUvjXGIBIP
```